components/scream/cime_config/namelist_defaults_scream.xml

```diff
@@ -0,0 +1,210 @@
+<?xml version="1.0"?>
+
+<!--
+
+The file that defines all parameter input files for SCREAM. This file
+will be processed by SCREAM's buildnml script and produces the
+$case/namelist_scream.xml which is the fully resolved/processed
+info from this file. We recommend users use atm-config-chg for
+tweaking their $case/namelist_scream.xml file.
+
+-->
+
+<namelist_defaults>
+
+  <!--
+    The following block defines convenience selectors. Case settings
+    can be used as selectors, but they are often not covenient due to
+    long names or having to process their values through regular expressions
+    to get the subpiece you are interested in. The convenience selectors
+    allow you to express this logic and capture it with a short name.
+
+    The follow attributes are how one defines a selector
+      name     : The name of the selector
+      case_env : The case value (thing you can xmlquery)
+      regex    : OPTIONAL, how to pick out the piece of the value you are interested in
+                 By default, the entire value is grabbed. If the regex does not match
+                 the case value, then the selector will evaluate to False.
+
+    Example using the nlev selector
+      <partmethod nlev="128">42</partmethod>
+      <partmethod nlev="72">420</partmethod>
+
+    Selector info will not be duplicated in the processed $case/namelist_scream.xml file.
+  -->
+  <selectors>
+    <selector name="hgrid" case_env="ATM_GRID"/>
+    <selector name="dyn" case_env="ATM_DYN_TARGET"/>
+    <selector name="nlev" case_env="SCREAM_CMAKE_OPTIONS" regex=".*SCREAM_NUM_VERTICAL_LEV ([0-9]+).*"/>
+  </selectors>
+
+  <!--
+      The following blocks define the input files for the model.
+      There are some rules that must be followed.
+
+      Lingo:
+        Outer element: An XML element that is not intended to be an input paramter,
+                       it is only for grouping/namespacing the elements within it
+        Parameter element: An XML element representing an individual input parameter.
+                           This element should not have child elements. It should have element text
+                           representing the desired value. These elements can have
+                           one or more selectors.
+        Selector: An XML attribute that checks a property of the case. See the documentation
+                  above the <selectors> element above for more details.
+        Metadata: An XML attribute that represents metadata for an input parameter
+
+      Rules:
+        1) Parameters can have any number of parameter elements. A parameter element
```

Comment:
Maybe change to "Parameters can have any number of parameter elements - one for each different value of a selector"? As written, it sounded a bit to me like "parameter elements can have any number of parameter elements"